lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -266,6 +283,40 @@ def resync(self, subscribers):
                 conn.close()
         self._on_ready.resync(subscribers)
 
+    def get_current_digest(self):
```

Comment:
Here and throughout your code: include Python type signatures

Our codebase doesn't do a good job of this yet, but that's because it was written before Python had type signatures 🙂 Python type signatures makes things significantly easier on future devs